tests/backup/backup_locked_bucket_test.go

```diff
@@ -412,3 +412,241 @@ var _ = Describe("{LockedBucketResizeOnRestoredVolume}", func() {
+				dash.VerifyFatal(preRuleStatus, true, "Verifying pre rule for backup")
+				preRuleNameList = append(preRuleNameList, ruleName)
+				postRuleStatus, ruleName, err := Inst().Backup.CreateRuleForBackup(appList[i], orgID, "post")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating post rule for deployed apps for %v with status %v", appList[i], postRuleStatus))
+				dash.VerifyFatal(postRuleStatus, true, "Verifying post rule for backup")
+				postRuleNameList = append(postRuleNameList, ruleName)
+			}
+		})
+		Step("Creating cloud credentials", func() {
+			log.InfoD("Creating cloud credentials")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, provider := range providers {
+				credName = fmt.Sprintf("%s-%s-%v", "cred", provider, time.Now().Unix())
+				cloudCredUID = uuid.New()
+				cloudCredUIDMap[cloudCredUID] = credName
+				err = CreateCloudCredential(provider, credName, cloudCredUID, orgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating cloud credentials %v", credName))
+			}
+		})
+		Step("Creating a locked bucket and backup location", func() {
+			log.InfoD("Creating locked buckets and backup location")
+			for _, provider := range providers {
+				for _, mode := range modes {
+					bucketName := fmt.Sprintf("%s-%s-%s", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
+					backupLocation = fmt.Sprintf("%s-%s-%s-lock", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
+					err := CreateS3Bucket(bucketName, true, 3, mode)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creating locked s3 bucket %s", bucketName))
+					BackupLocationUID = uuid.New()
+					backupLocationMap[BackupLocationUID] = backupLocation
+					err = CreateBackupLocation(provider, backupLocation, BackupLocationUID, credName, cloudCredUID,
+						bucketName, orgID, "")
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creating backup location %s", backupLocation))
+				}
+			}
+		})
+		Step("Register cluster for backup", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, "Creating source and destination cluster")
+			clusterStatus, err = Inst().Backup.GetClusterStatus(orgID, SourceClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+		})
+		Step("Create schedule policy after initializing volume resize", func() {
+			log.InfoD("Create schedule policy after initializing volume resize")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
```

Comment:
Use "log.FailOnError" at all places where we are verifying err against nil value

Reply:
Thanks!